tuxemon/resources/db/locale/eo.json

```diff
@@ -0,0 +1,179 @@
+{
+    "combat_run": "${{user}} forkuris!",
+    "combat_player_run": "Vi forkuris!",
+    "combat_used_x": "${{name}} estis uzata de ${{user}}!",
+    "combat_used_x_on_y": "${{name}} estis uzata al ${{target}} de ${{user}}!",
```

Comment:
To me, this sentence seems slightly unnatural and is awkward. Partly because of the passive, partly because of using 'al' instead of 'kontraŭ'.

How would something like
"${{user}} uzis ${{name}}n kontraŭ ${{target}}!".

That's assuming that ${{name}} is a normal noun ending in -o. So I don't actually know whether this would work :)